### ISSUE TITLE
Workspace sync error

### ISSUE THREAD

**Issue by N0N4M3pl** [warmuuh/milkman#121]
After creating new workspace with git sync options (sync using ssh key), i have problem with synchronization.
![milkman - workspace sync error](https://user-images.githubusercontent.com/7310909/128305777-5f42f294-6be9-4a8a-b5f9-51b9898c4a77.jpg)

In workspace i have created one grpc request, in folder, in collection:
![milkman - workspace sync struct](https://user-images.githubusercontent.com/7310909/128306407-15c246e2-835a-486c-8d7f-eaa4db8a9bba.jpg)

If You need some additional info - pls write, i will try to help you.

**Comment by warmuuh** [comment]
hi. are you sure that your plugins and your milkman version match? did you tried to install latest versions of both?

**Comment by N0N4M3pl** [comment]
I just resolved problem.

I'm using windows and choco install.
After upgrade milkman to latest version i have still using old binnaries (from version 5.0.0).

I was upgrading, reinstalling using force etc byt still binnaries not changed.
I looked into choco logs and found that it cant delete old files and ignore this during reinstall.

So i removed all choco cache connected with milkman and install it as clean as posiible.
Now i have real version 5.3.0 with 5.3.0 jars, etc.

(during my response  u write question :-) )
